Rickymac68 patch 2 (#18)
Cardano-Atlantic-Council/Minutes: 20250207.md

### Meeting Minutes

**Cardano Atlantic Council Meeting**

**Date:** February 7, 2025 18:00 UTC

**Attendees:** 
- Beatrice Anihiri
- Thiago Nunes
- Adam Dean
- Mike Hornan
- Jenny Brito
- Richard McCracken

**Purpose:** 

> Vote decision to replace the Interim Constitution with Constitution signed in Buenos Aires. Discuss operating costs for Cardano Atlantic core activities. How to make decisions when an emergency situation is present.

Reviewer: ```suggestion
- [x] Vote decision to replace the Interim Constitution with Constitution signed in Buenos Aires. 
- [x] Discuss operating costs for Cardano Atlantic core activities. 
- [x] How to make decisions when an emergency situation is present.
```

**Meeting Points:**

1. **Replace the Interim Constitution with the proposed Constitution GA**
 - All team members voted Yes verbally
 - Sufficient time has been allowed to process
 - Sufficient debate among the community has been observed

2. **How to make decisions when an emergency situation is present. (disaster)**
 - Catastrophic scenario requiring immediate vote:

> > Are we equipped and prepared to handle emergency vote  situations?

Reviewer: ```suggestion
> Are we equipped and prepared to handle emergency vote situations?
```

> Rapid building and distribution of body files.

> > Must have at least 3 of 6 signatures.

Reviewer: ```suggestion
> Must have at least 3 of 6 signatures. (Approval of at least 4 of 6 members is still required per bylaws)
```

> Establish must be done by time.
> Cross training of team members on alternate procedures in case of missing members during a short notice emergency.
 
3. **Scenarios: Must vote by the end of an epoch to stop a catastrophic change from executing**
 - Must vote to counter an attack by bad actor. (DDoS, economic attack).

>  - Must vote during a parameter change to put it back to it’s original value before it executes.

Reviewer: ```suggestion
 - Must vote during a parameter change to put it back to it's original value before it executes.
```

> What happens if a vote has to occur during a long lived network partition
> Who do we consider a valid source for emergency notification?
 - Valid SMEs to submit emergency reports. Proof of Life and Proof of emergency condition. Must use video and direct contact verification.
 - Network Halt - SPOs recover the network first.
 - Network Partition - SME assessment.

4. **Red Flag Response**
> 1. We become aware.

> > 2. Prove liveness or the source.

Reviewer: ```suggestion
> 2. Prove liveness of the source.
```

> 3. Second source other than the report source.
> 4. Verify red flag scenario is valid using testing when possible.
> 5. Independent SME to verify red flag is valid.
> 6. Members of CAC team verify red flag is valid.
> 7. Additional considerations.

> 5. **Opoerating costs and compensation for committee work**

Reviewer: ```suggestion
5. **Operating costs and compensation for committee work**
```

 - What is the actual value of time PER HOUR and level effort to operate as a constitutional committee member?
> Example: A typical average financial analyst in the United States is about $40/hour.

>  - What activities count as “must have” to perform the required duties of ICC?

Reviewer: ```suggestion
 - What activities count as "must have" to perform the required duties of ICC?
```

 - What activities count as important but not absolutely necessary?
 - How much time is required for socialization of issues, community feedback, and training?

>  - What is an appropriate amount to compensate reasonably without turning a withrawal request into a grift?

Reviewer: ```suggestion
 - What is an appropriate amount to compensate reasonably without turning a withdrawal request into a grift?
```

>  - Cost of hardware and legal requirements tallied in the operating budget.

Reviewer: ```suggestion
 - Cost of hardware, auditing, and legal requirements tallied in the operating budget.
```

> Example calculation: 5 to 10 hours per week per person, x 6 people, at about $40 per hour for typical US financial analyst pay. 5 hours per week under normal conditions and possibly 10 hours per week if a large number of proposals are active in parallel.

>  - CAC will not accept funds from Intersect, Pragma, or other entities. We will attempt to get the funding by requesting a treasury withdrawal action. 

Reviewer: ```suggestion
 - CAC will not accept funds from Intersect, Pragma, or other entities. We will attempt to get the funding by submitting our own budget and requesting an independent treasury withdrawal action. 
```

- When CAC submits an operating budget action we will immediately vote abstain on our own action.

**Action Items:**
1. Construct Yes votes on the Constitution.
2. All members sign - target submit February 12.

> 3. Calculate the average price of Ada over the last 4 years to present.

Reviewer: ```suggestion
3. Calculate the average price of Ada over the last 4 years to present.
4. Prepare a draft budget for socialization amongst the community.
```
